sql/core/src/main/scala/org/apache/spark/sql/execution/Exchange.scala

```diff
@@ -181,7 +180,17 @@ case class Exchange(newPartitioning: Partitioning, child: SparkPlan) extends Una
         }
       }
     }
-    new ShuffledRowRDD(rddWithPartitionIds, serializer, part.numPartitions)
```

Comment:
This is just some plumbing to allow our custom `UnsafeShuffledRowRDD` to be used.